lib/propolis/Cargo.toml

```diff
@@ -47,6 +53,24 @@ git = "https://github.com/oxidecomputer/omicron"
 branch = "main"
 optional = true
 
+[dependencies.p4rs]
```

Comment:
For ease of reading, can we include a comment here mentioning that these are associated w/ falcon too?

Reply:
Done